src/modules/displayMeals.js

```diff
@@ -1,12 +1,15 @@
 import fetchMeals from './fetchMeals.js';
 import renderMeals from './renderMeals.js';
+import selectCard from './popup.js';
 
 const displayMeals = async (url) => {
   // fetch data
   const data = await fetchMeals(url);
-
+  console.log(data);
```

Comment:
Delete console.log after finishing the project it makes your code ugly and it code shows important data on the console.

Delete any unused code it makes your code more readable and clean.